### PR TITLE
Allow custom particles for the plants

### PR DESCRIPTION
**Pull request by Ketheroth**

### CODE REVIEW

File: src/main/java/com/agricraft/agricore/plant/AgriParticleEffect.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.agricraft.agricore.plant;
+
+public class AgriParticleEffect {
+
+    private final String id;
+    private final double delta_x;
+    private final double delta_y;
+    private final double delta_z;
+    private final double probability;
```
Comment (InfinityRaider) [inline src/main/java/com/agricraft/agricore/plant/AgriParticleEffect.java:13]: Also add an array for growth stages.

**Review by InfinityRaider — CHANGES_REQUESTED** [review]
These will need to be added to the existing jsons I suppose (should be automatic with write back)

**Comment by Ketheroth** [comment]
I will make a PR for AgriPlant with the new JSONs when these PRs are good to be merged

**Review by InfinityRaider — APPROVED** [review]

**Comment by InfinityRaider** [comment]
Thanks!

**Pull request merged**